backend/src/schema/types/type/User.gql

```diff
@@ -1,173 +1,177 @@
 type User {
-  id: ID!
-  actorId: String
-  name: String
-  email: String! @cypher(statement: "MATCH (this)-[:PRIMARY_EMAIL]->(e:EmailAddress) RETURN e.email")
-  slug: String!
-  avatar: String
-  coverImg: String
-  deleted: Boolean
-  disabled: Boolean
-  disabledBy: User @relation(name: "DISABLED", direction: "IN")
-  role: UserGroup!
-  publicKey: String
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-30T11:29:10Z" title="Friday, August 30th 2019, 1:29:10 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -152,6 +154,7 @@ type Query {
   ): [User]
 
   blockedUsers: [User]
+  currentUser: User
```


What is this? :slightly_frowning_face:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T11:46:43Z" title="Friday, August 30th 2019, 1:46:43 pm +02:00">Aug 30, 2019</time>_

---


This seems to be a reminder from earlier tryouts …